rllib/examples/bandit/tune_lin_ucb_train_recommendation.py

```diff
@@ -95,7 +95,7 @@
     # Analyze cumulative regrets of the trials
     frame = pd.DataFrame()
     for result in results:
-        frame = frame.append(result.metrics_dataframe, ignore_index=True)
+        frame = pd.concat([frame, result.metrics_dataframe], ignore_index=True)
```

Comment:
what change caused this change?

Reply:
df.append has been deprecated and removed in e.g. pandas 1.5.x, which will be installed in some combinations of our requirements (1.3.5 is the last py3.7 release and still supported the deprecated API)